src/parse.ts

```diff
@@ -106,10 +106,10 @@ export function fixupTypes(types: MessageDefinition[]): void {
  * @returns - parsed message definition
  */
 export function parseRos2idl(messageDefinition: string): MessageDefinition[] {
-  return buildRos2IdlType(messageDefinition, ROS2IDL_GRAMMAR);
+  return buildRos2idlType(messageDefinition, ROS2IDL_GRAMMAR);
```

Comment:
Why the change in casing? It does seem like the `i` in `idl` should be capital? Or does lowercase match what we do for `msg` ?

Reply:
i mostly did it just from `ros2idl`, which is the schema name. I thought it would be more consistent, but I could be convinced otherwise

Reply:
Let's move this function to a separate file